tests/romtools/test_trial_space.py

```diff
@@ -5,6 +5,11 @@
 import romtools.trial_space.utils as utils
 
 
+def local_tensor_to_matrix(tensor_input):
```

Comment:
I think we should just call this `_tensor_to_matrix`

The underscores says that it's just for this module unless the "local" is referring to parallel/serial or something.

Reply:
It is because this is a test. I can use _ here too

Reply:
I don't understand.  What does `local` communicate?

Reply:
local to this test file , i will change it

Reply:
@fnrizzi I don't see an _impl? Am I understanding it correct that you stuck with the _ and then added the local functions to the test?

Reply:
yes i used a single _ for the functions inside the library, and then just use a copy of the function in this test file